core/src/main/java/org/apache/carbondata/core/util/path/CarbonTablePath.java

```diff
@@ -48,9 +48,11 @@
   protected static final String CARBON_UPDATE_DELTA_EXT = ".updatedelta";
   protected static final String DATA_PART_PREFIX = "part-";
   protected static final String BATCH_PREFIX = "_batchno";
-  protected static final String INDEX_FILE_EXT = ".carbonindex";
   protected static final String DELETE_DELTA_FILE_EXT = ".deletedelta";
 
+  public static final String INDEX_FILE_EXT = ".carbonindex";
```

Comment:
Can we change it to package private static member?
And can you change the above member from line 36 to 51 also

Reply:
ok